subsystems/win/basesrv/dosdev.c

```diff
@@ -809,7 +809,7 @@ CSR_API(BaseSrvDefineDosDevice)
         /* There's a target provided - new device, update buffer */
         else
         {
-            RtlInitUnicodeString(&LinkTarget, &CurrentBuffer[-TargetLength - 1]);
+            RtlInitUnicodeString(&LinkTarget, &CurrentBuffer[-(SSIZE_T)TargetLength - 1]);
```

Comment:
This is fine, thought I'd probably write it as `CurrentBuffer - TargetLength - 1`